build.gradle

```diff
@@ -14,7 +14,7 @@ def ENV = System.getenv()
 class Globals {
 	static def baseVersion = "0.10.1"
 	static def mcVersion = "20w18a"
-	static def yarnVersion = "+build.1"
+	static def yarnVersion = "+build.9"
```

Comment:
I assume this is to simplify the process of the implementation since the mapped code exists there

Reply:
yes, the registry class isn't mapped in build 1